content/blog/2018/08/2018-08-13-simple-pull-request-plugin-final-evaluation.adoc

```diff
@@ -0,0 +1,208 @@
+    # Post section for "stage1". All Conditions which are available in Jenkins
+    # declarative pipeline are supported
+    post:
+      failure:
+        - sh: "scripts/hello"
+
+# Outer post section. Just like declarative pipeline.
+post:
+  always:
+    - sh: "scripts/hello"
+----
+
+=== Tasks completed in Coding Phase 3
+
+
+. Add unit tests, JenkinsRule tests link:https://issues.jenkins-ci.org/browse/JENKINS-52495[JENKINS-52495]
+* [*] Refactor snippet generator to extensions (link:https://issues.jenkins-ci.org/browse/JENKINS-52491[JENKINS-52491])
+. Release 1.0 (link:https://issues.jenkins-ci.org/browse/JENKINS-52519[JENKINS-52519])
+. Plugin overview blog post
+
+
+
+=== Coding Phase 3 tasks not completed
+
+. Support the “when” Declarative Pipeline directive (link:https://issues.jenkins-ci.org/browse/JENKINS-52520[JENKINS-52520])
+. Nice2have: Support hierarchical report types (link:https://issues.jenkins-ci.org/browse/JENKINS-52521[JENKINS-52521])
+. ATH tests link:https://issues.jenkins-ci.org/browse/JENKINS-52496[JENKINS-52496]
+. Automatic Workspace Cleanup when PR is closed (link:https://issues.jenkins-ci.org/browse/JENKINS-51897[JENKINS-51897])
+
+. Test Multi-Branch Pipeline features support:
+.. [] Support for webhooks (link:https://issues.jenkins-ci.org/browse/JENKINS-51941[JENKINS-51941])
+.. Check if trusted people have approved a pull request and start build accordingly (link:https://issues.jenkins-ci.org/browse/JENKINS-52517[JENKINS-52517])
+. Finalize documentation (link:https://issues.jenkins-ci.org/browse/JENKINS-52518[JENKINS-52518])
+
+https://issues.jenkins-ci.org/browse/JENKINS-52452[Phase 3 Jira Epic]
+
+
+=== Phase 3 evaluation presentation video
+
+
+Video:
+++++
+//<center>
+////  <iframe width="720" height="400" src="https://www.youtube.com/embed/tuTODhJOTBU?start=3229"
+////  frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+//</center>
+++++
+
+=== Phase 3 evaluation presentation slides
+
+++++
+//<center>
+//    <iframe width="720" height="400" frameborder="0"
+//      src="https://speakerdeck.com/player/e7073ad74ee941b0b604d403462da1d3"></iframe>
+//</center>
+++++
+
+=== How to reach me
+
+* Email: gautamabhishek46@gmail.com
+* Gitter room: link:https://gitter.im/Jenkinsci/simple-pull-request-job-plugin[]
+
+=== References
+
+* link:https://github.com/jenkinsci/simple-pull-request-job-plugin[Project repository]
+* link:https://jenkins.io/projects/gsoc/2018/simple-pull-request-job-plugin/[Project page]
+* link:https://gitter.im/jenkinsci/simple-pull-request-job-plugin?utm_source=share-link&utm_medium=link&utm_campaign=share-link[Gitter chat]
+* link:https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20Jenkins%20AND%20component%20%3D%20simple-pull-request-job-plugin[Bug Tracker]
+* link:https://github.com/gautamabhishek46/dummy[Demo Repository]
+* link:https://www.youtube.com/watch?v=tuTODhJOTBU&t=3229s[Phase 2 Presentation video](July 12, 2018)
```

Comment:
do you have the same references for phase 1?

also, maybe move these links above where you link to the phase 2 blog post